test/integration/action/test-util.js

```diff
@@ -24,3 +25,15 @@ export const isPortOpen = async port => {
     client.connect(port, 'localhost');
   });
 };
+
+export const killProcess = async pid => {
+  let terminated = false;
+  while (!terminated) {
+    try {
+      process.kill(pid);
+      await nap(500);
+    } catch (e) {
+      terminated = true;
+    }
```

Comment:
What if `process.kill()` is successful? Will it still throw an exception?

Reply:
If `process.kill()` is successful, then it will error on the next attempt because the process doesn't exist anymore. I couldn't find any other way to determine whether the process was actually dead. 

Tried `process.terminated`, tried `process.on(SIGINT)`/other event handlers. Those could probably work if I put in a `sleep` after they occur and then restarted `btcd`, but I wanted to avoid sleeps.

Reply:
Ah ok. Let's see if this helps.